libraries/Microsoft.Bot.Builder/SimplePropertyAccessor.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder
+{
+    public class SimplePropertyAccessor<T> : IPropertyAccessor<T>
+    {
```

Comment:
Why "SimplePropertyAccesssor"? Consider "PropertyAccessor" as the name.  

I'm trying to think of examples in the .Net Framework (or any other framework) that has this pattern, and can't think of any. 

Looking at the code here, and the corresponding interfaces, I don' think we need:
IPropertyAccessor, IPropertyAccessor<T>, and SimplePropertyAccessor.

I think *two* of those could be removed. Just the class here seems sufficient, and I don't think the interfaces are needed. They're adding complexity and increasing our surface area w/o seeming to add benefit. What am I missing?